lib/shape.lua

```diff
@@ -169,6 +172,42 @@ function Shape:tick()
 	while self.theta > tau do
 		self.theta = self.theta - tau
 	end
+	while self.theta < -tau do
+		self.theta = self.theta + tau
+	end
+	-- ARC 1
+	local delta_arc_note = self.delta_arc_note / 30 
+	if delta_arc_note > 0 then
+		delta_arc_note = math.floor(delta_arc_note)
+	else
+		delta_arc_note = math.ceil(delta_arc_note)
+	end
```

Comment:
I don't know if there's an easier way to express this in Lua? What we want is to round the value in the direction of 0 for both positive and negative numbers.

That works, though.